tribler.sh

```diff
@@ -27,7 +27,8 @@ if [ "$UNAME" = "Linux" ]; then
     }
 
     python2.7 $TRIBLER_SCRIPT
-
+elif [[ "$UNAME" = CYGWIN_NT* ]]; then
+    python $TRIBLER_SCRIPT
```

Comment:
None of them work for me, I just was trying random stuff, never seen that syntax before either.